From: Ian Kent <ikent@redhat.com>
Subject: [RHEL 5 PATCH] patch correction (second attempt) - autofs - fix 	panic on mount fail - missing autofs module
Date: Thu, 17 May 2007 13:07:43 +0800
Bugzilla: 240307
Message-Id: <1179378463.3891.16.camel@raven.themaw.net>
Changelog: [autofs] fix panic on mount fail - missing autofs module



Hi,

This patch corrects a mistake in patch
linux-2.6-autofs-fix-panic-on-mount-fail.patch in the current CVS tree
in response to bz 240307 reported buy Prarit Bhargava who noticed the
compile warning. It's not good to have CVS issue warnings when building
but our current supported autofs doesn't use this module so I don't
think it's serious.

The typo was "autofs_kill_sb" was "autofs4_kill_sb" in hunk 1 of
inode.c, an obvious mistake.

Ian

---
--- linux-2.6.18.noarch/fs/autofs/inode.c.fix-panic-on-mount-fail-correction	2007-05-17 12:57:06.000000000 +0800
+++ linux-2.6.18.noarch/fs/autofs/inode.c	2007-05-17 12:57:14.000000000 +0800
@@ -19,7 +19,7 @@
 #include "autofs_i.h"
 #include <linux/module.h>
 
-void autofs4_kill_sb(struct super_block *sb)
+void autofs_kill_sb(struct super_block *sb)
 {
 	struct autofs_sb_info *sbi = autofs_sbi(sb);
 	unsigned int n;


